resources/assets/typescript/post.ts

```diff
@@ -0,0 +1,62 @@
+import Utils from "./utils";
+
+const utils = new Utils(),
+      Lang = (<any>window).Lang || {};
+
+export default class Post {
+    /**
+     * Create control functionality for posts
+     * 
+     * @param postToggles Elements used to toggle hiding/showing a post
+     * @param postDeletes Elements used to delete a post
+     */
+    constructor(protected postToggles: NodeListOf<Element> = document.querySelectorAll('.post__toggle'),
+                protected postDeletes: NodeListOf<Element> = document.querySelectorAll('.delete a')) {
+        const _this = this;
+
+        if (postToggles.length > 0) {
+            utils.forEach(postToggles, function(i, toggle) {
```

Comment:
Does TS not support arrow function syntax? (`(i, toggle) => {}`). I can't remember off the top of my head. Not a big deal either way.

Reply:
It does, went ahead and swapped those out.